Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/MSBuildTemplates/SDKProjectTemplate.csproj.template

```diff
@@ -24,6 +26,10 @@
         <ProjectReference Include="..\..\..\ExternalLib\ExternalLib.csproj" />
       </ItemGroup>
   -->
-  
-  <Import Project="$(MSBuildProjectName).g.targets" />
+
+  <ItemGroup>
```

Comment:
Intentionally empty `ItemGroup`?

Reply:
Yeah, hoping that NuGet packages would get put here.